src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import block from 'bem-cn-lite';
```

Comment:
Instead raw lib use `block` from `src/components/utils/cn.ts`, that create class with prefix